app/assets/javascripts/backbone/views/group_modal_view.js.coffee

```diff
@@ -0,0 +1,60 @@
+class ELMO.Views.GroupModalView extends Backbone.View
+
+  el: 'body'
+
+  events:
+    'click .save': 'save'
+
+  initialize: (options) ->
+    @list_view = options.list_view
+    @mode = options.mode
+
+    # TODO: Remove once edit_group link uses url_builder
+    @edit_link = options.edit_link
+
+    if $('.group-modal').length
+      $('.group-modal').replaceWith(options.html)
+    else
+      $('body').append(options.html)
+
+    this.show()
+
+  serialize: ->
+    @form_data = $('.qing_group_form').serialize()
```

Comment:
Probably better to return this than set it as an instance variable. We can discuss why.